src/types/identity/index.ts

```diff
@@ -11,10 +12,16 @@ export type CreateIdentityOptions = {
 
 export type NewIdentityRequest = {
   strategy: IdentityStrategy;
-  messageSignature: string;
   options: CreateIdentityOptions;
+  walletType: EWallet;
+  messageSignature?: string;
 };
 
+export enum EWallet {
```

Comment:
Maybe just `Wallet` which means either one of those two ones.

Reply:
`Wallet` is already a class in `ethers` package so it will be a conflict with names.